PurchasesCoreSwift/Public/PurchaserInfo.swift

```diff
@@ -0,0 +1,319 @@
+//
+//  PurchaserInfo.swift
+//  PurchasesCoreSwift
+//
+//  Created by Madeline Beyl on 7/9/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCPurchaserInfo) public class PurchaserInfo: NSObject {
```

Comment:
I had trouble running jazzy on my M1...working on that, so for now consider comments not ready for review...

Reply:
did you get jazzy to work? lmk if you need a hand